.eslintrc.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
     {
       files: ['*.ts', '*.tsx'],
       rules: {
-        '@typescript-eslint/no-shadow': ['error'],
```

Comment:
what exactly does this do?

Reply:
Es hilflt das wenn ich ein String nicht alsl string deklariere weil ich das variabel halten mus 
`var a = 10`
--> dadurch kann ich das machen und es wird nicht andauernd als Error angezeigt